src/api/public/apidocs-new/paths/build_project_repository_arch_package_history.yaml

```diff
@@ -0,0 +1,35 @@
+get:
+  description: This endpoint returns the build history for a specific artifact
+  operationId: getBuildProjectRepositoryArchPackageHistory
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+  responses:
+    '200':
+      description:
+        Returns ok when the build status is ok
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/buildstatus.yaml'
```

Comment:
In this case, I think the schema is not `buildstatus` but `buildhistory`.

Reply:
So true